src/ray/util/logging.cc

```diff
@@ -291,11 +241,7 @@ void RayLog::StartRayLog(const std::string &app_name, RayLogLevel severity_thres
   severity_threshold_ = severity_threshold;
   app_name_ = app_name;
   log_dir_ = log_dir;
-#if defined(RAY_USE_GLOG) || defined(RAY_USE_SPDLOG)
-#ifdef RAY_USE_GLOG
-  google::InitGoogleLogging(app_name_.c_str());
-  int level = GetMappedSeverity(static_cast<RayLogLevel>(severity_threshold_));
-#endif
+#ifdef RAY_USE_SPDLOG
```

Comment:
Can we remove `RAY_USE_SPDLOG` as well?

Reply:
`
#ifdef RAY_USE_SPDLOG
typedef ray::SpdLogMessage LoggingProvider;
#else
typedef ray::CerrLog LoggingProvider;
#endif
`
If we don't define RAY_USE_SPDLOG, ray::CerrLog will be used. If we remove RAY_USE_SPDLOG, how to provide a default logging provider?

Reply:
I don't think `CerrLog` is used anymore. We can delete it as well.

Reply:
I am okay with just deleting glog to make iteration faster btw. (and we can remove Cerrlog in the separate PR)